bigquery_etl/dryrun.py

```diff
@@ -241,6 +241,8 @@ def is_valid(self):
             and error.get("code", None) in [400, 403]
             and "does not have bigquery.tables.create permission for dataset"
             in error.get("message", "")
+            or "Permission bigquery.tables.create denied on dataset"
+            in error.get("message", "")
```

Comment:
It looks like this is the new error message when we are trying to run `init.sql` files